ingestion/src/metadata/ingestion/connections/test_connections.py

```diff
@@ -72,7 +76,9 @@ def test_connection_steps(steps: List[TestConnectionStep]) -> str:
             step.function()
             test_connection_result.success.append(f"'{step.name}': Pass")
 
-        except Exception:
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.debug(f"{step.name}-{exc}")
```

Comment:
should we have this as warning / error ?

Reply:
L81

Reply:
Yes, warning makes sense here.